### ISSUE TITLE
1.6.7 osx scheduled transaction stuck at "SUBMITTING"

### ISSUE THREAD

**Issue by Yuripetusko** [paritytech/parity#5717]
Was going to participate in ICO today, created scheduled transaction but it was stuck at Submitting and there's no way to cancel it. ICO is over and it's still there :)
![screenshot 2017-05-30 17 05 33](https://cloud.githubusercontent.com/assets/666169/26592964/51fba2a0-455a-11e7-85f0-d829f0064c6c.png)

Also maybe unrelated but I see `Parity//v1.6.7-beta-e128418-20170518/x86_64-macos/rustc1.17.0
Unknown capability` at bottom left corner

**Comment by johnerfx** [comment]
Hi, 
BAT ICO didn't start yet. It starts at 3,798,640 block exactly as you scheduled. 
Use "WHEN ARE WE THERE" dapp form Parity applications tab to find out what time this block takes place approximately.

**Comment by Yuripetusko** [comment]
@johnerfx it wasn't BAT but Mysterium ICO, it was at 13:00 today and I did time scheduling, I think yo ucan see a small part of my other transaction that is scheduled for BAT, please ignore it

**Comment by johnerfx** [comment]
Yes, you are right, my bad.

**Comment by tomusdrw** [comment]
If you have a scheduled transaction from an account every other transaction you'll try to send from that particular account will run only after the first one was executed.
So the transaction you scheduled for BAT is blocking each subsequent transaction.

You can cancel BOTH of the transactions by running once with `parity --no-persistent-txqueue` flag or using this config file:
https://paritytech.github.io/parity-config-generator/#config=eyJtaW5pbmciOnsibm9fcGVyc2lzdGVudF90eHF1ZXVlIjp0cnVlfSwiX19pbnRlcm5hbCI6eyJwbGF0Zm9ybSI6Ik1hYyBPUyJ9fQ==

And restarting Parity (remove the config file afterwards)

**Comment by Yuripetusko** [comment]
@tomusdrw this is it! I wish I knew about it before Mysterium ICO finished :) but alteast I will be prepared for next one. Thank you

**Comment by thebalaa** [comment]
This seems like a bug at worst and bad UX at best.

**Comment by tomusdrw** [comment]
@thebalaa This is a technical limitation, transaction has a fixed nonce (order) that needs to be defined at the moment of signing. 
I agree that UI should be more informative about this limitation.

**Comment by Yuripetusko** [comment]
Would be nice to have ability to cancel transactions that are in queue. At the moment it is not possible

**Comment by tomusdrw** [comment]
The PR is pending #5716
